### PR TITLE
8236917: TestInstanceKlassSize.java fails with "The size computed by SA for java.lang.Object does not match"

### PR DESCRIPTION
**Pull request by dhanalla**
backport of c0dce25756911ce49e7d8b034d5f12a25fc3c81d
8236917: TestInstanceKlassSize.java fails with "The size computed by SA for java.lang.Object does not match"
1. Clean backport from tip
2. validated the modified test case

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8236917](https://bugs.openjdk.org/browse/JDK-8236917) needs maintainer approval

### Issue
 * [JDK-8236917](https://bugs.openjdk.org/browse/JDK-8236917): TestInstanceKlassSize.java fails with "The size computed by SA for java.lang.Object does not match" (**Bug** - P3 - Approved)

### Reviewers
 * [Martin Doerr](https://openjdk.org/census#mdoerr) (@TheRealMDoerr - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk11u-dev.git pull/2712/head:pull/2712` \
`$ git checkout pull/2712`

Update a local copy of the PR: \
`$ git checkout pull/2712` \
`$ git pull https://git.openjdk.org/jdk11u-dev.git pull/2712/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 2712`

View PR using the GUI difftool: \
`$ git pr show -t 2712`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk11u-dev/pull/2712.diff">https://git.openjdk.org/jdk11u-dev/pull/2712.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk11u-dev/pull/2712#issuecomment-2116111725)

### CODE REVIEW

**Comment by dhanalla** [comment]
Adding a comment to keep this PR active.

**Review by TheRealMDoerr — CHANGES_REQUESTED** [review]
Sorry for the delay. I had started a review and forgot to finish it.

File: test/hotspot/jtreg/serviceability/sa/TestInstanceKlassSizeForInterface.java [hunk 1]
```diff
@@ -38,6 +38,7 @@
 import jdk.test.lib.process.OutputAnalyzer;
 import jdk.test.lib.SA.SATestUtils;
 import jdk.test.lib.Utils;
+import java.util.Arrays;
```
[the remark this answers is not in the document]
Comment (dhanalla) [reply test/hotspot/jtreg/serviceability/sa/TestInstanceKlassSizeForInterface.java]: @TheRealMDoerr, Thanks for reviewing the PR.

**Review by TheRealMDoerr — APPROVED** [review]
Thanks!

**Pull request closed without merging**